adapters/clients/replication.go

```diff
@@ -98,12 +98,12 @@ func (c *replicationClient) DigestObjectsInTokenRange(ctx context.Context,
 	}
 
 	var resp replica.DigestObjectsInTokenRangeResp
-	err = c.do(c.timeoutUnit*20, req, body, &resp)
+	err = c.do(c.timeoutUnit*20, req, body, &resp, 9)
```

Comment:
**question (non-blocking):** shall it be `numRetries` ?

Reply:
Good question, if I follow where this function call is used, I end up here: https://github.com/weaviate/weaviate/blob/nathanwilk7-prototyping-usecases-replica-coordinator-pull/adapters/repos/db/shard_hashbeater.go#L364 , which doesn't look like it uses Pull so I believe the best way to preserve current behavior is to hardcode the same current number of retries (9).